src/components/date-picker/date-picker.st.css

```diff
@@ -22,36 +21,51 @@
     dropdownWidth: 330px;
     dropdownArrowWidth: 11px;
     dropdownArrowHeight: 12px;
-    iconWidth: 36px;
+    iconSize: 16px;
+    iconOffset: 37px;
+    inputHeight: 36px;
```

Comment:
why here instead of the import?

Reply:
Removed. I had it set to 37px I think, but changed it back.